pallets/aleph/Cargo.toml

```diff
@@ -34,3 +34,4 @@ std = [
     "pallet-balances/std",
     "pallet-session/std"
 ]
+try-runtime = ["frame-support/try-runtime"]
```

Comment:
_inb4, I see that you did this the same way as Substrate does, but_ 

I'm wondering whether we should declare here `pallet-balances/try-runtime` and `pallet-session/try-runtime` as well

Reply:
nope, it is only important to specify them in the Cargo.toml of the runtime

Reply:
would it break something or is this just a redundancy in the wider context?